workshops/templates/base_without_navbar.html

```diff
@@ -4,9 +4,12 @@
   <head>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css" />
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
```

Comment:
There's one char missing at the end: `/`. The line should look like `<link rel="stylesheet" href="..." />`, otherwise it makes a lot of noise in the tests.